README.md

```diff
@@ -95,6 +95,35 @@ We support and actively test against certain third-party clients to ensure compa
 - `IFNULL(expr1, expr2)`: If expr1 is not NULL, IFNULL() returns expr1; otherwise it returns expr2.
 - `NULLIF(expr1, expr2)`: Returns NULL if expr1 = expr2 is true, otherwise returns expr1.
 
+## Configuration
+
+The behaviour of certain parts of go-mysql-server can be configured using either environment variables or session variables.
+
+Session variables are set using the following SQL queries:
+
+```sql
+SET <variable name> = <value>
+```
+
+### Memory joins
+
+- `EXPERIMENTAL_IN_MEMORY_JOIN`: if this environment variable is set it will perform all joins in memory.
```

Comment:
maybe this has less sense having MAX_MEMORY_INNER_JOIN. Shall we remove them?

Reply:
This ensures all of them are performed in memory without having to worry about which number to set. I would remove the `EXPERIMENTAL_` though